app/Models/Permission.php

```diff
@@ -42,22 +50,35 @@ class Permission extends Model
      */
     protected $guarded = ['id', 'created_at', 'updated_at'];
 
-     /**
-      * Cast values to correct type.
-      *
-      * @var array
-      */
-     protected $casts = [
-         'user_id' => 'integer',
-         'server_id' => 'integer',
-     ];
+    /**
+     * Cast values to correct type.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'subuser_id' => 'integer',
+    ];
 
+    /**
+     * Find permission by permission node.
```

Comment:
I call permission's a "permission node", pretty sure thats what you were referring to here.

If I make them static I have to call them statically, which I do not do.